Filters/exceptions.txt

```diff
@@ -23,7 +25,7 @@
 @@||ac.ebis.ne.jp^|
 ! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/602
 ! Don't block clicks in AWS emails
-@@||email.awscloud.com^
+@@||email.awscloud.com^|
```

Comment:
what blocks them?

Reply:
IDK about `email.awscloud.com`, just noticed it's lacking `|`. `analytics.edgekey.net` is in Spyware and EasyPrivacy, already whitelisted for contents filter in https://github.com/AdguardTeam/AdguardFilters/commit/c16daf177f26e00bf7ccd8c01195e0f8684f405e

Reply:
email.awscloud.com is not blocked by our DNS

![image](https://user-images.githubusercontent.com/8361299/126977269-0914c181-50a2-40c8-8263-a86b884f7809.png)

Reply:
@ameshkov https://github.com/AdguardTeam/AdGuardSDNSFilter/commit/53ce5d14788a6e99c2d5cab42963b5bba6664017